### PR TITLE
Rename name references

### LINKED ISSUE #4: Rename old names

### PR DESCRIPTION
**Pull request by kiootic**
fix #4

### CODE REVIEW

File: packages/authgear-react-native/android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="io.skygear.reactnative">
+          package="io.authgear.reactnative">
```
Comment (louischan-oursky) [inline packages/authgear-react-native/android/src/main/AndroidManifest.xml:2]: Our domain is authgear.com so the package should be `com.authgear.reactnative`.

File: packages/authgear-react-native/ios/AGAuthgearReactNative.h [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #import <React/RCTBridgeModule.h>
 #import <React/RCTEventEmitter.h>
 
-@interface SGSkygearReactNative : RCTEventEmitter <RCTBridgeModule>
+@interface AGAuthgearReactNative : NSObject <RCTBridgeModule>
```
Comment (louischan-oursky) [inline packages/authgear-react-native/ios/AGAuthgearReactNative.h:8]: Why do we change the parent class?

File: packages/authgear-react-native/android/src/main/java/io/authgear/reactnative/AuthgearReactNativeModule.java [hunk 3]
```diff
@@ -135,7 +135,7 @@ public void getAnonymousKey(String kid, Promise promise) {
             header.putString("kid", kid);
             header.putString("alg", "RS256");
 
-            String alias = "io.skygear.keys.anonymous." + kid;
+            String alias = "io.authgear.keys.anonymous." + kid;
```
Comment (louischan-oursky) [inline packages/authgear-react-native/android/src/main/java/io/authgear/reactnative/AuthgearReactNativeModule.java:138]: `com.authgear.`

File: packages/authgear-react-native/android/src/main/java/io/authgear/reactnative/AuthgearReactNativeModule.java [hunk 4]
```diff
@@ -162,7 +162,7 @@ public void signAnonymousToken(String kid, String data, Promise promise) {
         }
 
         try {
-            String alias = "io.skygear.keys.anonymous." + kid;
+            String alias = "io.authgear.keys.anonymous." + kid;
```
Comment (louischan-oursky) [inline packages/authgear-react-native/android/src/main/java/io/authgear/reactnative/AuthgearReactNativeModule.java:165]: `com.authgear.`

File: packages/authgear-react-native/ios/AGAuthgearReactNative.m [hunk 5]
```diff
@@ -244,7 +244,7 @@ + (BOOL)application:(UIApplication *)application
       kid = [[NSUUID UUID] UUIDString];
     }
 
-    NSString *tag = [@"io.skygear.keys.anonymous." stringByAppendingString:kid];
+    NSString *tag = [@"io.authgear.keys.anonymous." stringByAppendingString:kid];
```
Comment (louischan-oursky) [inline packages/authgear-react-native/ios/AGAuthgearReactNative.m:247]: Here as well.

File: packages/authgear-react-native/ios/AGAuthgearReactNative.m [hunk 6]
```diff
@@ -271,7 +271,7 @@ + (BOOL)application:(UIApplication *)application
 {
   if (@available(iOS 10.0, *)) {
     NSData *data = [s dataUsingEncoding:NSUTF8StringEncoding];
-    NSString *tag = [@"io.skygear.keys.anonymous." stringByAppendingString:kid];
+    NSString *tag = [@"io.authgear.keys.anonymous." stringByAppendingString:kid];
```
Comment (louischan-oursky) [inline packages/authgear-react-native/ios/AGAuthgearReactNative.m:274]: Here as well.

File: packages/authgear-web/src/container.ts [hunk 7]
```diff
@@ -24,7 +24,7 @@ export interface ConfigureOptions {
   endpoint: string;
   /**
    * isThirdPartyApp indicate if the application a third party app.
-   * A third party app means the app doesn't share common-domain with Skygear Auth thus the session cookie cannot be shared.
+   * A third party app means the app doesn't share common-domain with Authgea thus the session cookie cannot be shared.
```
Comment (louischan-oursky) [inline packages/authgear-web/src/container.ts:27]: typo

File: packages/authgear-react-native/src/index.ts [hunk 8]
```diff
@@ -233,7 +233,7 @@ export class ReactNativeContainer<
 }
 
 /**
- * Default Skygear APIs container.
+ * Default Authgear APIs container.
```
Comment (louischan-oursky) [inline packages/authgear-react-native/src/index.ts:236]: Make it simply `Default container.`

File: packages/authgear-react-native/ios/AGAuthgearReactNative.h (hunk 2 above)
Comment (kiootic) [reply packages/authgear-react-native/ios/AGAuthgearReactNative.h:8]: Inheriting `RCTEventEmitter` needs to implements `supportedEvents`, which is no longer needed and removed in a previous commit. So just revert to use a plain `NSObject`.

**Comment by kiootic** [comment]
@louischan-oursky Fixed. When merging, please use a format like `feat: xxx` so enable generation of changelog.

**Comment by louischan-oursky** [comment]
Great! Any more detailed guideline I can read somewhere else?

**Comment by louischan-oursky** [comment]
The android src directory also need rename from `main/io/authgear` to `main/com/authgear`.

**Comment by kiootic** [comment]
I planned to setup change log generation with this tool: https://github.com/git-chglog/git-chglog

It matches the commit message with a regex to filter & extract parts of commit messages, then use Go's template (customizable) to generate the change log.

**Pull request merged**